Makefile

```diff
@@ -42,6 +43,8 @@ $(WEAVER_EXE) $(WEAVEDNS_EXE): common/*.go
 
 $(WEAVER_EXE): router/*.go weaver/main.go
 $(WEAVEDNS_EXE): nameserver/*.go weavedns/main.go
+$(SIGPROXY_EXE): sigproxy/main.go
+	go build -o $@ ./$(shell dirname $@)
```

Comment:
I added another `go build` instruction here rather than add `$(SIGPROXY_EXE)` to the target above because the netgo check fails when compiling the signal proxy - possibly because it doesn't import net at all? Should we do this a different way?